src/Microsoft.Azure.WebJobs.Extensions.Kafka/Diagnostics/ActivityTags.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka
```

Comment:
Same as above

Reply:
Namespaces help resolve the conflict between two classes (with the same name) in different namespaces.
Also, every class in this extension is in the same namespace "Microsoft.Azure.WebJobs.Extensions.Kafka".